README.md

```diff
@@ -49,11 +50,11 @@ packages and certificates.
 
 #### Packages
 
-For URLs that look like `https://example.com/wpk/url/to/amp.html`, the frontend
+For URLs that look like `https://example.com/htxg/url/to/amp.html`, the frontend
 server must internally reverse-proxy these requests to something like:
 
 ```
-http://packager.internal/priv-amppkg/doc?fetch=https%3A%2F%2example.com%2Furl%2Fto%2Famp.html&sign=https%3A%2F%2Fexample.com%2Furl%2Fto%2Famp.html
+http://packager.internal/priv/doc?fetch=https%3A%2F%2example.com%2Furl%2Fto%2Famp.html&sign=https%3A%2F%2Fexample.com%2Furl%2Fto%2Famp.html
```

Comment:
nit: it looks 'F' before the first 'example.com' got dropped

?fetch=https%3A%2F%2example.com  -->
?fetch=https%3A%2F%2Fexample.com

Reply:
Wow, good catch. Fixed.